### DISCUSSION TITLE
DC Amps - Entity is non numeric

### DISCUSSION THREAD

**Discussion by george369** [WillCodeForCats/solaredge-modbus-multi#344] [Q&A]
Entity becomes unavailable, displaying "entity is non-numeric" error, when there is no production from the panels. Issue happens only with the below specified entity while all other entities (specifically DC Volts and DC Power) display zero as result (Likewise for AC). Entity is used in HA standard gauge card. When solar production is resumed, entity and eventually card, behave normally. 

Entity: sensor.solaredge_i1_dc_current

Any ideas?

Thank you all in advance!

**Comment by WillCodeForCats** [comment]
This is not a problem I've seen personally; when my inverters are off/sleep DC current reads zero.

You will need to upload your diagnostic file snapshot at the exact time the error is occurring.

**Reply by george369** [reply]
[config_entry-solaredge_modbus_multi-8d9e852de49436303eeb3d98b6a3c9d8.json.txt](https://github.com/WillCodeForCats/solaredge-modbus-multi/files/11873034/config_entry-solaredge_modbus_multi-8d9e852de49436303eeb3d98b6a3c9d8.json.txt)


<img width="216" alt="github" src="https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/44071284/dcb955b0-fe88-4515-9671-b14a8e4d17c2">

<img width="330" alt="github2" src="https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/44071284/7c02505b-fe85-4895-a963-bf5f808beff3">

**Reply by WillCodeForCats** [reply]
Thanks, I will try simulating this on my test platform to see if there's a better way to handle the values the inverter is returning.

**Reply by george369** [reply]
Thank you very much for your efforts, much appreciated.
